libs/coin-framework/src/api/types.ts

```diff
@@ -6,15 +6,19 @@ export type BlockInfo = {
 
 export type Operation = {
   hash: string;
+  // TODO: check if useful
   address: string;
   type: string;
+  // TODO: check if we want Asset array
   value: bigint;
+  // TODO: check if we want Asset array
   fee: bigint;
   block: BlockInfo;
   senders: string[];
   recipients: string[];
   date: Date;
-  transactionSequenceNumber: number;
+  // TODO: check if useful
+  transactionSequenceNumber?: number;
   // Field containing dedicated value for each blockchain
   details?: Record<string, unknown>;
```

Comment:
If I am reading it correctly, `details` will contain the `tokenId` ?

I think we may also need the token type (TRC10 or TRC20), and contract address, if I understand it correctly `tokenId` only uniquely identifies TRC10 tokens ?

Reply:
Yes it makes sense to me. An operation can have a single token transfer ?

Reply:
done